### PR TITLE
feat: old blogs.

### PR DESCRIPTION
**Pull request by amylas**

### CODE REVIEW

File: _posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "2018-11-30-Reading: How the words Class-solidification becoming popular"
+date:   2018-11-30 19:31:07 +0800
+categories: readingnote
+---
+
+[Article Link](2018-11-30-Reading: The-dilemma-of-Class-solidification)
```
Comment (u9520107) [inline _posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown]: Please state your sources more clearly. If it's a website, please include the URL to the article, if it's a book, perhaps include the author.

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful"
+date:   2018-12-02 19:31:07 +0800
+categories: readingnote
+---
+
+[Article Link](2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful)
```
Comment (u9520107) [inline _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: Similarly please include a detailed reference to the article you read, I am sure there will be people who are interested in reading that too.

File: _posts/2018-12-19-presentation: some thinking.markdown [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+Sorry for the late presentation for which I would like to blame the procrastination of humanity. There are something I would like to talk about after the last 2 weeks' work, after think through.
```
Comment (u9520107) [inline _posts/2018-12-19-presentation: some thinking.markdown]: I am not quite sure what you are trying with this post.
Is this a summary or reflection on a presentation that you've saw? Or you just want to write some life reflections on the events you've gone through? I think you should clearly state the purpose of this writing and why you are sharing this.

File: _posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown (hunk 1 above)
Comment (amylas) [reply _posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown]: It's a TED I watched, I have attached the video link.

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown (hunk 2 above)
Comment (amylas) [reply _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: t's a TED I watched and I just attached the video link.

File: _posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown (hunk 1 above)
Comment (amylas) [reply _posts/2018-11-30-Reading: How the words Class-solidification becoming popular.markdown]: It'a a viewpoint from a book named Homo Deus, A Brief History of Tomorrow. I just add it to the article.

File: _posts/2018-12-19-presentation: some thinking.markdown (hunk 3 above)
Comment (amylas) [reply _posts/2018-12-19-presentation: some thinking.markdown]: I have deleted that blog which it's suppose write to myself.

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+2、You are averse to change.
+3、You're not willing to take risks.
+4、You believe you deserve success based on credentials.
+5、You constantly go after whatever's exciting at the moment.
+6、You can't commit to a decision.
+7、You don't believe in yourself.
+
+#### Reflection:
+The video is really refreshing and I think there are really some points I need to improve.
+First, make new friends especially from different area, have different life experience or have different ideas. Start by say hi to strangers, welcome different ideas, inspects, perspective etc.
```
Comment (u9520107) [inline _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: First, make new friends```,``` especially from different ```areas```, ```with``` different ```experiences``` or different ideas. Start by ```saying``` hi to strangers, ```welcoming``` different ideas, ```inspirations``` (not sure what you mean by inspects, maybe inspirations?), and perspectives.

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Second, go to different public occasion, try out new things, go to have much different life experience.
```
Comment (u9520107) [inline _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: ```Secondly```, go to different public ```events```, try out new things, ```and``` go ```experience``` a different life style.

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Third, take risks, dont fear risks, when you take more responsibility it always means you will learn more and able to take more responsibility. Look at now from 10 years later, you will always grateful for the biggest risk you have taken, so have more courage now, dont be feared, there are really nothing to fear.
```
Comment (u9520107) [inline _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: ```Thirdly```, take risks ```and don't``` fear risks```. When``` you take more responsibility```,``` it ```means you will always learn more``` and ```be``` able to take ```even``` more responsibility. ```Looking back in 10 years time```, you will ```be``` grateful for the biggest ```risks``` you have taken. So ```be more courageous``` now, ```don't be afraid``` (feared has a totally different meaning), there are really nothing to fear.

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+Start doing what supposed to do, and finish what you have started.
```
Comment (u9520107) [inline _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: Start doing what ```you are``` supposed to do and finish what you have started. (The comma in the middle is not necessary)

File: _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+So hardworking, embrace change, take risks, believe in youself and start doing.
```
Comment (u9520107) [inline _posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown]: So ```work hard```, embrace change, take risks, believe in yourself```,``` and start ```acting```. ("start doing" is not a phrase that people use, you could say "start doing something", but I think just "start acting" is probably sufficient)

**Pull request closed without merging**